vtk_viewer/test/utest.cpp

```diff
@@ -32,15 +31,13 @@ TEST(ViewerTest, TestCase1)
   points2->InsertNextPoint(pt2);
   points2->InsertNextPoint(pt3);
   points2->InsertNextPoint(pt4);
-  
+
   vtkSmartPointer<vtkPolyData> cut = vtkSmartPointer<vtkPolyData>::New();
   cut = vtk_viewer::cutMesh(data, points2, false);
 
   cout << "cutter points: " << cut->GetPoints()->GetNumberOfPoints() << "\n";
   cout << "cutter lines: " << cut->GetNumberOfLines() << "\n";
 
-
-
   vtk_viewer::VTKViewer viz;
   std::vector<float> color(3);
 
```

Comment:
@trentwall please make suggested changes, and ping me for a final review before merge

Reply:
I have finished all the changes that you and mat requested.